lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +207,29 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const container = resource.spec.url?.replace('oci://', '');
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: container,
+        };
+        if (resource.spec.ref?.digest) {
+          dep.currentDigest = resource.spec.ref.digest;
+          dep.currentValue = 'latest';
+          dep.datasource = DockerDatasource.id;
+        } else if (resource.spec.ref?.tag) {
+          const containerTag = getDep(container + ':' + resource.spec.ref?.tag);
+          dep.autoReplaceStringTemplate =
+            '{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}';
```

Comment:
```suggestion
            '{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}';
```
i don't think we want to have the depName inside the tag

Reply:
Fixed in upcoming commit

Reply:
please reduce noise and read our contributing guide[^1]

[^1]: https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting